soap/src/java/com/zimbra/soap/mail/message/FileSharedWithMeRequest.java

```diff
@@ -105,6 +112,7 @@ public FileSharedWithMeRequest(String action, String granteeId, String fileName,
         this.rights = rights;
         this.contentType = contentType;
         this.size = size;
+        this.ownerAccountId = ownerAccountId;
```

Comment:
Had quick question:
User1 shares the file with User2, and user2 shares the same file with user3, by looking at the change it seems now we are passing the file's original owner id right? now how user3 will see it shared by "User2" or "User1"?

Reply:
User 3 will see file shared by user1 , and this same information goes into email notification as well

Reply:
@gabhinavTI do you mean the account ID will be for User 1 to access the file Or the email that is received will show the files share by User1 to User3?

Reply:
1) user1 accountID will be used to access files in format `accountId:fileId`, earlier we were passing user2  in this scenario hence it was failing
2) File owner shown at user3 end would be user User1 - attached the first image
3) Email received shows the file was shared by User1 - attached the second image

Further clarifications for screenshot:
`abhinav1 `shares folder to `abhinav7` and further `abhinav7` shares file from shared folder to `abhinav8`
Ultimately `abhinav1` is the owner and `abhinav8` is a grantee. The same is in email , same on File information

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/96303121/193204921-5a0e9d8a-a759-4373-a0fd-0f8d428a725f.png">

<img width="1786" alt="image" src="https://user-images.githubusercontent.com/96303121/193205076-8fe79832-64eb-42a5-9c65-8ef4c875b08c.png">

Reply:
Email should show the file shared from abhinav7 to abhinav8 not from abhinav1 to abhinav8.

Reply:
This works the same way for shared folders too. Email content change will be taken care separately for displaying the sharee & sharer info about the share.

Reply:
have we created a backlog story to track this?

Reply:
yes, ZCS-12375